example-app/SantokuApp/src/bases/ui/picker/PickerContainer.test.tsx

```diff
@@ -49,23 +44,47 @@ describe('PickerContainer only with required props', () => {
     expect(sut).toMatchSnapshot('Before animation started');
     expect(animatedView).toHaveAnimatedStyle({transform: [{translateY: 1000}]});
 
+    /*
+      eslint-disable-next-line @typescript-eslint/no-floating-promises --
+      If advancing a timer changes the state of a component, the timer must be run within an act.
+      However, since act is `Thenable`, ESLint will issue a warning if you do not do something like await.
+      For convenience, disable the relevant rule in this file.
+     */
     startAnimation();
 
     // アニメーション中は`transform`が変化すること
+    /*
+      eslint-disable-next-line @typescript-eslint/no-floating-promises --
+      If advancing a timer changes the state of a component, the timer must be run within an act.
+      However, since act is `Thenable`, ESLint will issue a warning if you do not do something like await.
+      For convenience, disable the relevant rule in this file.
+     */
```

Comment:
```
    act(() => {
      jest.advanceTimersByTime(x);
    });
```
部分を関数化して `no-floating-promises` は1回だけ書くのがキレイそう？

Reply:
こんなイメージ？
```typescript
const nonThenableAct = (msToRun: number) => {
  /*
    eslint-disable-next-line @typescript-eslint/no-floating-promises --
    If advancing a timer changes the state of a component, the timer must be run within an act.
    However, since act is `Thenable`, ESLint will issue a warning if you do not do something like await.
    For convenience, disable the relevant rule in this file.
   */
  act(() => {
    jest.advanceTimersByTime(msToRun);
  });
};
```

Reply:
そんな感じです。

関数名だけ `時間を進める` みたいなものが良いと思います。
（`advanceTimersByTime` をコピーするのが自然？）

act 要素も合ったほうが良いなら追加で。

Reply:
そもそも、`await`すれば`eslint-disable`しなくても良いのでは？という意見も挙がった

Reply:
以下を理由に、actは`await`するように修正しましたー
（`eslint disable`することによって、このコードをあまり理解せずにコピーして使用した際に、本来`await`すべき所を`await`しないでテスト実施する可能性もあるため）
88864c109803d7dbc8adaf9d72b120f417b80d0a